### DISCUSSION TITLE
'known bugs' section in documentation

### DISCUSSION THREAD

**Discussion by jablka** [ics-py/ics-py#391] [Ideas]
hello,
the thread regarding '_Datetimes are converted to UTC at parsing time_' appears closed now.
https://github.com/ics-py/ics-py/issues/188

So I assume the bug is fixed?
So perhaps the '_known bugs_' section in documentation should be updated:
https://icspy.readthedocs.io/en/stable/misc.html#datetimes-are-converted-to-utc-at-parsing-time

**Comment by N-Coder** [comment]
Yes, the documentation is pretty outdated with version 0.8, this is still one of the big points on the [roadmap](https://github.com/ics-py/ics-py/issues/245). Feel free to open a PR improving things!
